Resources/public/scss/modules/universal-discovery/_tab.nav.item.scss

```diff
@@ -13,8 +13,7 @@
 
     &:hover,
     &:focus {
-        background: $ez-ground-base-dark-hover;
-        border-radius: calculateRem(8px) calculateRem(8px) 0 0;
+        background: $ez-ground-base-medium-hover;
```

Comment:
I'd also suggest to harmonize tab icons color (fill):
- Normal state, I'd suggest to add harmonized fill color to `.c-tab-nav-item__icon` => `fill: $ez-color-base-medium;`
- Hover state: Add to `.c-tab-nav-item--selected, .c-tab-nav-item--selected:hover, .c-tab-nav-item--selected:focus` harmonized fill color `.c-tab-nav-item__icon {fill: $ez-black;}`